docs/Documentation/run_RBM.md

```diff
@@ -1,66 +1,64 @@
+In the example, we run hydrological model **VIC4.2.d** to generate flow and energy fluxes, which are input data for RBM. The example global parameter files can be found in the directory **run_VIC**. To create the necessary files (executable file to run VIC), you need to download [VIC4.2.d](https://github.com/UW-Hydro/VIC/releases/tag/VIC.4.2.d). The instructions to generate the executable file **vicNl** and run VIC can be found [here](http://vic.readthedocs.io/en/vic.4.2.d/). Copy the executable file **vicNl** to the folder **run_VIC**. The example global files to generate baseflow, runoff and heat budgets are also directly included in the folder **run_VIC**, which you can use directly. The two were implemented as follows:
 <center>**./vicNl –g global_param_Salmon_0.5_flux**</center>
 which generates base flow and runoff
 
 and
 <center>**./vicNl –g global_param_Salmon_0.5_full_data** </center>
 which generates heats budget
 
-PS: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
+Note: The output directories need to be specified in each global parameter files, **global_param_Salmon_0.5_flux** and **global_param_Salmon_0.5_full_data** in this case.
 
 ### Step 2 Generate topology file
-In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl**, (copied from ***../Perl_Scripts***) using the direction file that was created as described in the VIC model development (see [VIC model](https://uw-hydro.github.io/code/)). All the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. The number of basins and sub-basins in a river system is limited only by the amount of computer memory, but there must be only a single outlet. For example, the Columbia River system could be modeled in its entirety. Modeling the Columbia River system and another river system, for example, the Fraser River, would require two separate simulations. Also, simulated river system cannot contain braided networks. The following example is from the Salmon River in Idaho:
+In the folder, ***../Test_Case***, run the perl script, **build_network_beta.pl** (copied from ***../Perl_Scripts***) using the direction file. Example direction file **Salmon_flowd_0.5.dir** can be found in ***../Test_Case***. In the flow direction file, all the grid cells surrounding the basin of interest must contain a negative one (-1) for purposes of determining the headwaters segments. Also, a simulated river system cannot contain braided networks. The following example is from the Salmon River in Idaho:
```

Comment:
For second comment
1) Fixed
2) I change that sentence to: `Also, we cannot simulate a river system containing braided networks.`
3) Fixed